openshift-ci/Dockerfile.registry.upstream.dev

```diff
@@ -9,7 +13,11 @@ COPY ${OLM_SOURCE} /registry/performance-addon-operator-catalog
 RUN find /registry/performance-addon-operator-catalog/ -type f -exec sed -i "s|REPLACE_IMAGE|${FULL_OPERATOR_IMAGE}|g" {} \; || :
```

Comment:
another option is getting rid of this entirely, possibly enhancing our `csv-replace-imageref` tool and/or packing in a different image

Reply:
The problem, not just a find or sed binaries, but the call to the `RUN` at all, because under the neat `RUN` calls to `/bin/sh -c`.

But to remove this line can make our Dockerfile cleaner, so I definitely not versus it, but it should be part of the different PR :)

Reply:
ok, but if so every `RUN` instance will fail, not just this one, right?

Reply:
Yes, but to avoid it I just copy `sh` from /usr/bin to /bin